### PR TITLE
[doc] fix typo in WebSocketServer description

### PR DESCRIPTION
**Pull request by tniessen**

### CODE REVIEW

**Comment by GITHUBGUY12357999** [comment]
ok let me chk.

File: doc/ws.md [hunk 1]
```diff
@@ -91,7 +91,7 @@ must be provided or an error is thrown. An HTTP server is automatically created,
 started, and used if `port` is set. To use an external HTTP/S server instead,
 specify only `server` or `noServer`. In this case the HTTP/S server must be
 started manually. The "noServer" mode allows the WebSocket server to be
-completly detached from the HTTP/S server. This makes it possible, for example,
```
Comment (GITHUBGUY12357999) [inline doc/ws.md:94]: Fix this line 94.

**Review by GITHUBGUY12357999 — CHANGES_REQUESTED** [review]
Fix line number 94 thank you.

**Comment by tniessen** [comment]
The comments by @GITHUBGUY12357999 here and in other repositories appear to be spam.

**Comment by GITHUBGUY12357999** [comment]
@tniessen I  got red marked codes so I want you to change them.

**Comment by GITHUBGUY12357999** [comment]
We can close this,

 thank you the issue is resolved.

**Comment by lpinca** [comment]
Thank you.

**Comment by GITHUBGUY12357999** [comment]
Ur Most welcome.

**Pull request merged**